tests/foreman/ui_airgun/test_syncplan.py

```diff
@@ -473,3 +522,58 @@ def test_positive_synchronize_custom_product_weekly_recurrence(
         repo_values = session.repository.read(product.name, repo.name)
         for repo_type in ['Packages', 'Errata', 'Package Groups']:
             assert repo_values['content_counts'][repo_type] > '0'
+
+
+@tier3
+def test_positive_synchronize_custom_product_custom_cron_past_sync_date(
+        session, module_org):
+    """Create a sync plan with past datetime as a sync date,
+    add a custom product and verify the product gets synchronized
+    on the next sync occurrence based on custom cron interval
+
+    :id: 4d9ed0bf-a63c-44de-846d-7cf302273bcc
+
+    :expectedresults: Product is synchronized successfully.
+
+    :BZ: 1279539
+
+    :CaseLevel: System
+    """
+    interval = 60 * 60  # 'hourly' sync interval in seconds
+    delay = 5 * 60
+    plan_name = gen_string('alpha')
+    product = entities.Product(organization=module_org).create()
+    repo = entities.Repository(product=product).create()
+    with session:
+        startdate = (
```

Comment:
I am not sure that logic here is right.
You have interval `*/5 * * * *` which means every 5th minute
so it is fine to have 
```
delay = 5 * 60
...
stardate = session.browser.get_client_datetime()`
...
```

Reply:
you are correct! will update accordingly

Reply:
I suggest to add `real time` scenario as it is actually reproduce real user experience. And then we can add whatever we can imagine